pom.xml

```diff
@@ -29,6 +29,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
 		<geotools.version>15.4</geotools.version>
+		<spring.security.jwt>1.0.9.RELEASE</spring.security.jwt>
```

Comment:
Does this file need a Licensing header?

Reply:
None of our pom.xmls have them. I'm not sure how nice the XML comments would be when formatted. We can probably live without them in this file.